ee/tabby-webserver/src/schema/auth.rs

```diff
@@ -384,6 +385,9 @@ pub trait AuthenticationService: Send + Sync {
         code: String,
         client: Arc<GithubClient>,
     ) -> std::result::Result<GithubAuthResponse, GithubAuthError>;
+
+    async fn smtp_credentials(&self) -> Result<Option<SMTPInfo>>;
+    async fn set_smtp_credentials(&self, creds: SMTPInfo) -> Result<()>;
```

Comment:
Let's focus db changes in this PR. service interface can be done in a follow up PR.